acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -13,9 +13,6 @@
                 'ubuntu' => 'apache2',
 }
```

Comment:
I meant confine the platforms to only those that we provide package names for. As in
`confine :to, :platform => /el|centos|fedora|debian|sles|ubuntu/`